server/modules/analysis/repository/chain/update.js

```diff
@@ -0,0 +1,28 @@
+import * as pgPromise from 'pg-promise'
+
+import { TableChain } from '../../../../../common/model/db'
+
+import { db } from '../../../../db/db'
+
+/**
+ * Updates the common script of the given processing chain.
+ *
+ * @param {!string} surveyId - The survey id.
+ * @param {!string} chainUuid - The processing chain uuid.
+ * @param {!string} script - The script content.
+ * @param {pgPromise.IDatabase} [client=db] - The database client.
+ *
+ * @returns {Promise<any[]>} - The result promise.
```

Comment:
use params and params.surveyId ... 
return ```Promise<null>```

Reply:
sure